kube-client/src/client/auth/mod.rs

```diff
@@ -360,9 +360,14 @@ fn token_from_gcp_provider(provider: &AuthProviderConfig) -> Result<ProviderToke
     // Command-based token source
     if let Some(cmd) = provider.config.get("cmd-path") {
         let params = provider.config.get("cmd-args").cloned().unwrap_or_default();
-
+        let drop_env = provider.config.get("cmd-drop-env").cloned().unwrap_or_default();
```

Comment:
This is basically just a freestyle thing from the upstream https://github.com/kubernetes/client-go/blob/7697067af71046b18e03dbda04e01a5bb17f9809/tools/clientcmd/api/types.go#L167-L172 right?

I'm OK with it, but would add a comment that it's a non-standard property that's not used in client-go/kubectl if there's no upstream equivalent for it.